__tests__/workers/workers.ts

```diff
@@ -0,0 +1,39 @@
+import {
+  personalizedDigestWorkers,
+  workers as infraWorkers,
+} from '../../.infra/workers';
```

Comment:
This won't work currently since jest will start importing files and dependencies from `.infra` and we don't install those in tests. 😢

Reply:
Ah, right. The deps will then be requested to be installed. Let me think of another way.

Reply:
I found a workaround to make the importing from the test work, though not sure if it is ideal. Though at least it tests all subscriptions.